pydicom/dataelem.py

```diff
@@ -166,6 +166,14 @@ def __init__(self,
             tag = Tag(tag)
         self.tag = tag
         self.VR = VR  # Note!: you must set VR before setting value
+        self.VR = VR
+        if self.VR == 'UN':
```

Comment:
You've got the same line twice

Reply:
Should we be changing the VR for all the elements if its explicit VR encoded? Can you just decode the Specific Character Set value if VR == UN and its python 3?

Reply:
> You've got the same line twice

Ah, I accidentally did this in the wrong branch and after stashing and changing the branch did an error in conflict resolving...